pom.xml

```diff
@@ -257,7 +257,24 @@
       <version>${token-support.version}</version>
       <scope>test</scope>
     </dependency>
-
+    <dependency>
+      <groupId>no.nav.security</groupId>
+      <artifactId>token-validation-spring-test</artifactId>
+      <version>${token-support.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.squareup.okhttp3</groupId>
```

Comment:
jeg kan ikke si at jeg er stor tilhenger av å lempe på med biblioteker som ikke er støttet av spring-boot... er dette for en WireMock implementasjon)?

anbefaler å se på spring-boot sin egen implementasjon av wiremock og som er støttet fra spring-boot-test

Reply:
Det er for nav sitt nye spring validation. Det kreves pga en bug i biblioteket som ikke er fikset enda. Tror det har noe med Java versjon å gjøre som skaper feilen. Må da importere nyere versjon av okhttp3 for at det skal fungere